Tasks/SshV0/Tests/L0.ts

```diff
@@ -1,16 +1,170 @@
-import fs = require('fs');
 import assert = require('assert');
+import tmrm = require('azure-pipelines-task-lib/mock-test');
 import path = require('path');
 
-describe('SshV0 Suite', function () {
-    before(() => {
+function runValidations(validator: () => void, tr, done) {
+    try {
+        validator();
+        done();
+    }
+    catch (error) {
+        console.log("STDERR", tr.stderr);
+        console.log("STDOUT", tr.stdout);
+        done(error);
+    }
+}
+
+describe('SshV0 Suite', function() {
+    this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT) || 20000);
+
+    it('Fails for missing endpoint', (done) => {
+        delete process.env['sshEndpoint'];
+        delete process.env['commands'];
+        delete process.env['runOptions'];
+
+        let tp = path.join(__dirname, 'L0SshRunner.js');
+        var tr = new tmrm.MockTestRunner(tp);
+
+        tr.run();
+
+        runValidations(() => {
+            assert(tr.invokedToolCount == 0, 'should not have run any tools');
+            assert(tr.failed, 'task should have failed');
+            assert(tr.stdout.indexOf('Input required: sshEndpoint') >= 0, 'wrong error message: "' + tr.stdout + '"');
+        }, tr, done);
     });
 
-    after(() => {
+    it('Fails when user name is not provided in the endpoint', (done) => {
+        delete process.env['runOptions'];
+        process.env['sshEndpoint'] = 'IDUserNameNotSet';
+        process.env['commands'] = 'ls -l';
+
+        let tp = path.join(__dirname, 'L0SshRunner.js');
+        var tr = new tmrm.MockTestRunner(tp);
+
+        tr.run();
+
+        runValidations(() => {
+            assert(tr.invokedToolCount == 0, 'should not have run any tools');
+            assert(tr.failed, 'task should have failed');
+            assert(tr.stdout.indexOf('Endpoint auth data not present: IDUserNameNotSet') >= 0, 'wrong error message: "' + tr.stdout + '"');
+        }, tr, done);
     });
 
-    it('Does a basic hello world test', function(done: MochaDone) {
```

Comment:
did we lose this test? Was it not doing anything worthwhile?

Reply:
Yeah, it was a placeholder